src/auth/HomePrivate.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Outlet, Navigate } from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { RootState } from 'states/store';
+import AdminHomePage from 'components/admin-MainPage/AdminHomePage';
+import AdminNoGroupPage from 'components/admin-MainPage/AdminNoGroupPage';
+import OnBoardingPage from 'pages/OnBoardingPage';
+
+const HomePrivate = () => {
+  const loginState = useSelector((state: RootState) => state.login);
+  const isLogin: boolean = loginState.islogin;
+  const isAdim: boolean = loginState.userData.isAdmin;
+  const isGroup: boolean = loginState.userData?.groupName.length > 0;
+
+  return (
+    <>
+      {isLogin ? (
+        isAdim ? (
+          isGroup ? (
+            <AdminHomePage />
+          ) : (
+            <AdminNoGroupPage />
+          )
+        ) : isGroup ? (
+          <div>알바홈</div>
+        ) : (
+          <div>매니저에게 초대받으세요</div>
+        )
+      ) : (
+        <OnBoardingPage />
+      )}
+    </>
+  );
```

Comment:
렌더링될 컴포넌트가 어떤 것일지 한 눈에 들어오지 않아서 약간 정리하면 좋을 것 같아요!

```suggestion
  const isLogin: boolean = loginState.islogin;
  
  if(!isLogin) {
    return <OnBoardingPage />
  }
  
  const isAdim: boolean = loginState.userData.isAdmin;
  const isGroup: boolean = loginState.userData?.groupName.length > 0;
  
  return (
    <>
      {isAdmin && isGroup && <AdminHomePage />}
      {isAdmin && !isGroup && < AdminNoGroupPage />}
      
      {!isAdmin && isGroup && <div>알바홈</div>}
      {!isAdmin && !isGroup && <div>매니저에게 초대받으세요</div>}
    </>
  );
```